war/src/main/scss/modules/dialogs.scss

```diff
@@ -0,0 +1,83 @@
+$jenkins-dialog-padding: 1rem;
+
+.jenkins-dialog {
+  border-radius: 0.6rem;
+  border: none;
+  background: transparent;
+  box-shadow: var(--modal-box-shadow);
```

Comment:
Still uses some definitions from the modals as they are already supported by the black theme.